tests/lib/apt.sh

```diff
@@ -2,6 +2,9 @@
 
 . $TESTSLIB/pkgdb.sh
 
+curr_history=/var/log/apt/history.log
```

Comment:
Can you please make this nicer for other distributions. Put some structure to it so that we can have a similar implementation for zypper, dnf, yum and others.